tests/unit/test_UIARemoteOps/test_highLevel.py

```diff
@@ -0,0 +1,700 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2023 NV Access Limited
+
+"""
+Unit tests for the high-level UI Automation Remote Operations API.
+"""
+
+from unittest import TestCase
+from unittest.mock import Mock
+from ctypes import POINTER
+from UIAHandler import UIA
+from UIAHandler._remoteOps import operation
+from UIAHandler._remoteOps import remoteAPI
+from UIAHandler._remoteOps.lowLevel import (
+	PropertyId,
+)
+
+
+class TestHighLevel(TestCase):
```

Comment:
I think it would be valuable to split this suite up into operands e.g. test bool, test if, etc